shared/lib_shared_inverter.cpp

```diff
@@ -8,9 +9,93 @@ SharedInverter::SharedInverter(int inverterType, int numberOfInverters,
 	m_numInverters = numberOfInverters;
 	m_sandiaInverter = sandiaInverter;
 	m_partloadInverter = partloadInverter;
+	m_tempEnabled = false;
 }
 
-void SharedInverter::calculateACPower(const double powerDC_Watts, const double DCStringVoltage)
+bool sortByVoltage(const double i[], const double j[])
+{
+	return (i[0] < j[0]);
+}
+
+bool SharedInverter::setTempDerateCurves(double* curve1, double* curve2, double* curve3)
+{
+	if (curve1[0] < 0 || curve1[1] <= -99 || curve1[2] >= 0) return false;
+
+	// if multiple curves provided, partition operating V range

```

Comment:
Partitioning, or just sorting them to be in order?

Reply:
It was partitioning, but now that you mention it, interpolating might be better. What do you think?